app/templates/jinja2/dashboard/index.html

```diff
@@ -23,6 +23,90 @@
 <div class="ui-grid s-dashboard__game-info align-content-start">
   <div class="ui-grid align-content-start">
   
+  {% if emails.exclude(player_made) %}
+  {% call card("Mod Emails") %}
+        <table class="table table-striped">
+          <thead>
+          <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Date</th>
+            <th scope="col" />
+          </tr>
+          </thead>
+          <tbody>
+          {% for email in emails.exclude(player_made) %}
+            <tr>
+              <td data-toggle="tooltip" data-placement="top" title="{{ email.name }}">{{ email.name }}</td>
+              <td>{{ email.created_at }}</td>
```

Comment:
This line is erroring. I'm not sure by what, but the error received is the following:
![image](https://user-images.githubusercontent.com/17097380/91878467-56bbb300-ec4d-11ea-9187-8db1c2cdc100.png)

There is probably an error in here somewhere.